router.go

```diff
@@ -750,6 +750,12 @@ func (p *ControllerRegister) ServeHTTP(rw http.ResponseWriter, r *http.Request)
 			}
 			context.Input.CopyBody(BConfig.MaxMemory)
 		}
+
+		if r.ContentLength > BConfig.MaxFileSize {
```

Comment:
So if I upload a file with 129MB, does it fail?

Reply:
Yes.
It will return http status 413 if file size is over 128MB by default.

Reply:
So this behavior is not acceptable. For big files, we need to think about how to upload it by splitting into several parts.

Reply:
No, the "multipart" DOES NOT limit file size.
The OP wants limitation on file size and that is quite normal.